pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/JuniperKitRequest.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsm.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
+public class JuniperKitRequest {
+    //This class is to imagine what the request payload from Juniper will/should look like
+    private String firstName;
+    private String lastName;
+    private String street1;
+    private String street2;
+    private String city;
+    private String state;
+    private String postalCode;
+    private String country;
+    private String phoneNumber;
+    String juniperKitId;
```

Comment:
Could re rename `juniper*` to `external*`?  Today it's juniper, but the API could also be used by other systems.

Reply:
Gah never mind, I see we have `external` all over the place already...

Reply:
yeah... GBF...